doc/src/dev/tools.rst

```diff
@@ -0,0 +1,65 @@
+Tools
+=======
+
+rocTX
+------------
+MIGraphX driver provides `roctx` knob which can be used with `rocprof` binary to get marker timing information for each MIGraphX operator.  
+In order to help user to process timing information, rocTX helper script is provided at `/tools/roctx.py`.
```

Comment:
Write `tools/roctx/py` without the first slash.

Reply:
Why is there a slash between `roctx` and `py`? 
Removed the first slash.